src/pyOpenMS/pxds/Fitter1D.pxd

```diff
@@ -13,8 +13,8 @@ cdef extern from "<OpenMS/TRANSFORMATIONS/FEATUREFINDER/Fitter1D.h>" namespace "
     cdef cppclass Fitter1D(DefaultParamHandler):
         # wrap-inherits:
         #  DefaultParamHandler
-        Fitter1D() nogil except +
-        Fitter1D(Fitter1D) nogil except +
+        Fitter1D() nogil except + # wrap-doc:Abstract base class for all 1D-dimensional model fitter
+        Fitter1D(Fitter1D) nogil except + # wrap-ignore
```

Comment:
See comment to # wrap-ignore above. Let's wait for Hannes comment and then check the other instances as well.

Reply:
ok